.github/workflows/e2e-build.yaml

```diff
@@ -8,6 +8,46 @@ on:
         type: string
 
 jobs:
+  build-remover:
+    name: "Build remover image for e2e tests"
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    steps:
+      - name: Harden Runner
+        uses: step-security/harden-runner@1f99358870fe1c846a3ccba386cc2b2246836776
+        with:
+          egress-policy: audit
+      - name: Set up Go 1.19
+        uses: actions/setup-go@v3
+        with:
+          go-version: 1.19
+      - name: Setup buildx instance
+        uses: docker/setup-buildx-action@v2
+        with:
+          use: true
+      - uses: actions/cache@v3.3.1
+        with:
+          key: ${{ runner.OS }}-go-${{ hashFiles('**/go.sum') }}
+          restore-keys: |
+            ${{ runner.os }}-go-
+          path: |
+            ~/go/pkg/mod
+            ~/.cache/go-build
+      - uses: crazy-max/ghaction-github-runtime@v2
+      - name: Check out code into the Go module directory
+        uses: actions/checkout@v3
+      - run: 'echo ${{ inputs.bucket-id }}'
+      - name: Set env
+        run: |
+          echo REMOVER_REPO=remover >> $GITHUB_ENV
+          echo REMOVER_TAG=test >> $GITHUB_ENV
+      - name: Build remover
+        run: 'make docker-build-remover OUTPUT_TYPE=type=oci,dest=./${REMOVER_REPO}_${REMOVER_TAG}.tar,name=${REMOVER_REPO}:${REMOVER_TAG}'
+      - name: Upload Build Artifacts
+        uses: actions/upload-artifact@v3
+        with:
+          name: ${{ inputs.bucket-id }}
+          path: remover_test.tar     
```

Comment:
This should replace `build-eraser`, but you've left both.

Reply:
we need to build both eraser and remover for the e2e tests (same as below)